packages-web/calendar/src/components/Calendar.ts

```diff
@@ -16,7 +16,8 @@ import "react-big-calendar/lib/css/react-big-calendar.css";
 import "../ui/Calendar.scss";
 import "../ui/CalendarLoader.scss";
 
-export const DragAndDropCalendar = withDragAndDrop(BigCalendar);
+const localizer = BigCalendar.momentLocalizer(moment);
```

Comment:
Is this constant being used in any other place or just for props?

Reply:
It is only used in the calendar class. Do you want me to define the constants inside the class?

Reply:
If is only used in one place, put inside

Reply:
Done